Formula/m/mumps.rb

```diff
@@ -0,0 +1,56 @@
+class Mumps < Formula
+  desc "MUltifrontal Massively Parallel sparse direct Solver"
+  homepage "https://mumps-solver.org/"
+  url "https://mumps-solver.org/MUMPS_5.6.2.tar.gz"
+  sha256 "13a2c1aff2bd1aa92fe84b7b35d88f43434019963ca09ef7e8c90821a8f1d59a"
+  license "CECILL-C"
+
+  # Core dependencies
+  depends_on "gcc"
```

Comment:
Does it depend on GCC at runtime?

Reply:
No, how do I express a build/link only dep?

Reply:
Thanks!